### ISSUE TITLE
Paket dotnet tool doesn't run on 3.0 previews

### ISSUE THREAD

**Issue by baronfel** [fsprojects/Paket#3634]
### Description

running `paket restore` on the 3.0 preview7 runtime fails with the following error:

```
It was not possible to find any compatible framework version
The specified framework 'Microsoft.NETCore.App', version '2.1.0' was not found.
  - The following frameworks were found:
      3.0.0-preview7-27912-14 at [/usr/share/dotnet/shared/Microsoft.NETCore.App]

You can resolve the problem by installing the specified framework and/or SDK.

The .NET Core frameworks can be found at:
  - https://aka.ms/dotnet-download
```

I think we need a build of the paket tool that multi-targets to 3.0, because the default roll-forward strategy only supports going up a minor version (ie from 2.1 to 2.2).

**Comment by baronfel** [comment]
Documentation about roll-forward behaviors is here: https://github.com/dotnet/core-setup/blob/master/Documentation/design-docs/roll-forward-on-no-candidate-fx.md#roll-forward-in-absence-of-candidate-fx

**Comment by baronfel** [comment]
setting `DOTNET_ROLL_FORWARD_ON_NO_CANDIDATE_FX=2` in my build container sidesteps the issue, but we should either have some documentation around this or publish a multi-targeted build soonish.

**Comment by baronfel** [comment]
Instead of the user having to set an environment variable, we could package a runtimesettings configuration file that opts-in to roll-forward on major versions: https://natemcmaster.com/blog/2019/01/09/netcore-primitives-3/#runtimeconfigtemplatejson

**Comment by matthid** [comment]
Same thing in fake: https://github.com/fsharp/FAKE/issues/2325

**Comment by baronfel** [comment]
I wondered why this felt like deja-vu. Would you like me to replicate this PR over there?

**Comment by matthid** [comment]
> Would you like me to replicate my PR over there?

Sure, would love to merge it :).
Didn't realize there was a fix for this which doesn't need upgrading the sdk

**Comment by baronfel** [comment]
Closing, was fixed in #3635
